tests/perf/actor_double_activation/test.js

```diff
@@ -66,3 +85,26 @@ export function handleSummary(data) {
     'stdout': JSON.stringify(data),
   };
 }
+
+// add disruption
+
+const selector = {
+  namespace: __ENV.TEST_NAMESPACE,
+  select: {
+    labels: {
+      testapp: __ENV.TEST_APP_NAME
+    }
+  }
+}
```

Comment:
Semi here

Maybe we should add prettier to the codebase...

Reply:
I think js formatting is less important here as this is used only for tests and generally speaking these files should not have more than 100 lines with no external imports/requires.

However, considering that the effort to add prettier config and a github action presumably is really small we should do that, can I do this later? Will create an issue to track this

Reply:
Yes, can you do that in a separate PR?

It's just because we're now adding a lot of JS code and some consistency would not be bad.

Reply:
Yes, I can do that, thanks!